src/utils/logger.ts

```
@@ -13,25 +88,158 @@ export class Logger {
+    ].join('\n');
+  }
+
+  /**
+   * Logs an error message
+   * @param message The message to log
+   * @param meta Optional metadata to include
+   */
+  static error(message: unknown, meta?: Record<string, unknown>): void {
+    const messageStr = typeof message === 'string' ? message : String(message);
+    console.error(this.formatMessage(messageStr, 'error'));
+    if (meta) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)